tensorflow/lite/toco/tflite/op_version.cc

```diff
@@ -49,11 +49,16 @@ string GetMinimumRuntimeVersionForModel(const Model& model) {
           {{OperatorType::kDepthwiseConv, 3}, "1.14.0"},
           {{OperatorType::kAdd, 1}, "1.5.0"},
           {{OperatorType::kAdd, 2}, "1.14.0"},
+          {{OperatorType::kAdd, 3}, "1.15.0"},
+          {{OperatorType::kAdd, 4}, kPendingReleaseOpVersion},
           {{OperatorType::kAddN, 1}, "1.14.0"},
           {{OperatorType::kSpaceToBatchND, 1}, "1.6.0"},
           {{OperatorType::kSpaceToBatchND, 2}, "1.14.0"},
           {{OperatorType::kSub, 1}, "1.6.0"},
           {{OperatorType::kSub, 2}, "1.14.0"},
+          {{OperatorType::kSub, 3}, "1.15.0"},
+          {{OperatorType::kSub, 4}, "1.15.0"},
+          {{OperatorType::kSub, 5}, kPendingReleaseOpVersion},
```

Comment:
I am not sure about this place: the runtime version was not set for other cases of SUB operator: like 5D version, old version with POT param scale. I added them as 1.15.0 and the new case: 16-bit + general case as kPendingReleaseOpVersion.

There are tests in the file operator_test.cc.
Should I add tests for my case ? I can do this in a separate PR - it will be easy to review. Please let me know.

Reply:
Testing separately seems fine.